maven/plan.ps1

```diff
@@ -9,11 +9,15 @@ $pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 $pkg_source="http://mirrors.gigenet.com/apache/${pkg_name}/maven-3/${pkg_version}/binaries/apache-maven-${pkg_version}-bin.zip"
 $pkg_shasum="7e6cfe98dc9c16ae6aa267db277860594695144d719c99d1fc519e89346a8edf"
 $pkg_dirname="apache-$pkg_name-$pkg_version"
+$pkg_bin_dirs=@("bin")
+$pkg_lib_dis=@("lib")
 
-function Invoke-SetupEnvironment {
-    Set-RuntimeEnv MAVEN_HOME "$pkg_prefix" [-force]
- }
-
+function Invoke-Prepare {
+    $env:JAVA_HOME = "$(Get-HabPackagePath corretto8)"
```

Comment:
This isn't a problem with your code, but we would like to mirror the Linux plan behavior where-ever possible. In this case, can you add a `Write-BuildLine "Setting JAVA_HOME=$env:JAVA_HOME"` line here? (Sorry, I copy-pasta'd from the Linux plan, please fix my syntax, if this is not the right powershell syntax!)

Reply:
I can do that real quick, I was misunderstood when you guys asked me not to use "Write-Host" i think.